packages/medusa/src/services/line-item.ts

```diff
@@ -138,14 +176,13 @@ class LineItemService extends BaseService {
 
   /**
    * Create a line item
-   * @param {LineItem} lineItem - the line item object to create
-   * @return {LineItem} the created line item
+   * @param {DeepPartial<LineItem>} lineItem - the line item object to create
+   * @return {Promise<LineItem>} the created line item
    */
-  async create(lineItem) {
+  async create(lineItem: DeepPartial<LineItem>): Promise<LineItem> {
```

Comment:
nit: To be consistent with what we've done in already converted services, we could define a `CreateLineItemProps`.